tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -63,87 +76,131 @@ def assertGeometryColumnsCount(self, expected_count):
             expected_count
         )
 
-    def test_add_gis_field(self):
-        """
-        Tests the AddField operation with a GIS-enabled column.
-        """
+    def assertSpatialIndexExists(self, table, column):
+        with connection.cursor() as cursor:
+            indexes = connection.introspection.get_indexes(cursor, table)
+        self.assertIn(column, indexes)
+
+    def alter_gis_model(self, migrationclass, modelname, fieldname, blank=False, fieldclass=None):
         project_state = self.set_up_test_model()
         self.current_state = project_state
-        operation = migrations.AddField(
-            "Neighborhood",
-            "path",
-            fields.LineStringField(srid=4326),
-        )
+        args = [modelname, fieldname]
+        if fieldclass:
+            args.append(fieldclass(srid=4326, blank=blank))
+        operation = migrationclass(*args)
         new_state = project_state.clone()
         operation.state_forwards("gis", new_state)
         with connection.schema_editor() as editor:
             operation.database_forwards("gis", editor, project_state, new_state)
         self.current_state = new_state
+
+    def test_add_geom_field(self):
+        """
+        Test the AddField operation with a geometry-enabled column.
+        """
+        self.alter_gis_model(migrations.AddField, 'Neighborhood',
+            'path', False, fields.LineStringField)
         self.assertColumnExists("gis_neighborhood", "path")
 
         # Test GeometryColumns when available
         if HAS_GEOMETRY_COLUMNS:
             self.assertGeometryColumnsCount(2)
 
+        # Test Spatial Indices when available
         if self.has_spatial_indexes:
-            with connection.cursor() as cursor:
-                indexes = connection.introspection.get_indexes(cursor, "gis_neighborhood")
-            self.assertIn('path', indexes)
+            self.assertSpatialIndexExists('gis_neighborhood', 'path')
+
+    @skipUnlessDBFeature("supports_raster")
```

Comment:
try to be consistent and use single quotes in most places (sorry existing code is inconsistent)